Extensions/ExtendedMath.json

```diff
@@ -0,0 +1,553 @@
+          "parameterObjects": "",
+          "useStrict": true
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Any integer",
+          "longDescription": "",
+          "name": "a",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Any integer",
+          "longDescription": "",
+          "name": "b",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Returns the lowest common multiple of two numbers ",
+      "fullName": "Lowest common multiple (lcm)",
+      "functionType": "Expression",
+      "name": "lcm",
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "function gcf(a, b) {\r\n    while (a != b) {\r\n        if (a > b) {\r\n            a = a - b;\r\n        } else {\r\n            b = b - a;\r\n        }\r\n    }\r\n    return a;\r\n}\r\n\r\nfunction lcm(a, b) {\r\n    return Math.abs(a * b) / gcf(a, b);\r\n}\r\n\r\nconst a = eventsFunctionContext.getArgument(\"a\");\r\nconst b = eventsFunctionContext.getArgument(\"b\");\r\n\r\nif (Number.isInteger(a) === false || Number.isInteger(b) === false) {\r\n    eventsFunctionContext.returnValue = 0;\r\n} \r\nelse if (a === 0) {\r\n    eventsFunctionContext.returnValue = b;\r\n} \r\nelse if (b === 0) {\r\n    eventsFunctionContext.returnValue = a;\r\n} else {\r\n    eventsFunctionContext.returnValue = lcm(a, b);\r\n}",
+          "parameterObjects": "",
+          "useStrict": true
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Any integer",
+          "longDescription": "",
+          "name": "a",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Any integer",
+          "longDescription": "",
+          "name": "b",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Returns the input multiplied by all the previous whole numbers",
```

Comment:
Returns the product of all positive integers less than or equal to n (the specified number).